.travis.yml

```diff
@@ -1,6 +1,9 @@
 notifications:
   email: false
 
+before_install:
+  nvm install 12
```

Comment:
In other projects we've used the `xenial` distribution on [travis](https://docs.travis-ci.com/user/reference/xenial/), which has Node `12.18.2` already.

Have you tried that or would you consider using that as opposed to updating node as part of the build?

Reply:
@jkeck Happy to try a newer distro. I do note that core Blacklight uses `bionic`. 

https://github.com/projectblacklight/blacklight/blob/master/.travis.yml#L1

Is there a specific preference for `xenial` for this, or should I match core BL?

Reply:
I'd say +1 to `bionic` which should give you the updated node automagically.

Reply:
Ah, I hadn't noticed that we weren't even defining a dist.  If Bionic gives us the right node, then 👍.

Reply:
@jkeck @mejackreed Bionic work great. Thanks!